### PR TITLE
docs: createPantherClient() first parameter

### PR DESCRIPTION
**Pull request by bechir**
This minor change is for improving the doc after facing the same issue as #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,20 @@ Since Panther implements the API of popular libraries, it already has extensive
 * For the `Crawler` class, read [the DomCrawler documentation](https://symfony.com/doc/current/components/dom_crawler.html)
 * For WebDriver, read [the PHP WebDriver documentation](https://github.com/php-webdriver/php-webdriver)
 
+### Changing a client host and port
```
Comment (dunglas) [inline README.md:256]:
```suggestion
### Changing the Hostname and Port of the Built-in Web Server
```

Comment (dunglas) [reply README.md:256]: Could you also move this new section after the list of the environment variables please?

File: README.md [hunk 2]
```diff
@@ -253,6 +253,20 @@ Since Panther implements the API of popular libraries, it already has extensive
...
+
+If you want to change the host and/or the port, pass the `hostname` and `port` to the `$options` parameter of the `createPantherClient()` method:
```
Comment (dunglas) [inline README.md:258]:
```suggestion
If you want to change the host and/or the port used by the built-in web server, pass the `hostname` and `port` to the `$options` parameter of the `createPantherClient()` method:
```

File: README.md [hunk 3]
```diff
@@ -253,6 +253,20 @@ Since Panther implements the API of popular libraries, it already has extensive
...
+```php
+// ...
+
+$client = self::createPantherClient([
+    'hostname' => 'localhost',
+    'port' => 8080
```
Comment (dunglas) [inline README.md:264]:
```suggestion
    'hostname' => 'example.com', // Defaults to 127.0.0.1
    'port' => 8080, // Defaults to 9080
```

File: README.md [hunk 4]
```diff
@@ -253,6 +253,20 @@ Since Panther implements the API of popular libraries, it already has extensive
...
+]);
+
+// ...
```
Comment (dunglas) [inline README.md:267]:
```suggestion
```

**Review by dunglas — CHANGES_REQUESTED** [review]
Thanks for documenting this! I made some minor comments.

File: README.md (hunk 1 above)
Comment (bechir) [reply README.md:256]: Ok

**Comment by bechir** [comment]
I reviewed the code

**Comment by dunglas** [comment]
Thanks @bechir

**Pull request merged**